perses/annihilation/ncmc_switching.py

```diff
@@ -1216,20 +1268,22 @@ def __init__(self, temperature, system, functions, nsteps=0, steps_per_propagati
         gamma = collision_rate
 
         # NCMC variables
-        self.addGlobalVariable('lambda', 0.0) # initial total energy (kinetic + potential)
-        self.addGlobalVariable('total_work', 0.0) # initial total energy (kinetic + potential)
+        self.addGlobalVariable('lambda', 0.0) # parameter switched from 0 <--> 1 during course of integrating internal 'nsteps' of dynamics        self.addGlobalVariable('lambda', 0.0) # alchemical switching parameter
+        self.addGlobalVariable('total_work', 0.0) # cumulative total work in kT
+        self.addGlobalVariable('shadow_work', 0.0) # cumulative shadow work in kT
```

Comment:
It is not necessary, but it is useful. For example, for the benzene <--> naphthalene system, we could analyze the effects of omitting the shadow work for VVVR. We could also quantitatively say how much GHMC helps the NCMC acceptance rate.